python/src/testing_module.i

```diff
@@ -26,5 +26,7 @@
 #include "openturns/OTtestcode.hxx"
 %}
 
+%include OTtestcode_doc.i
```

Comment:
Shouldn't the `DistributionChecker` be in the `experimental` module?

Reply:
it belongs with the testing stuff like assert_almost_equal